### DISCUSSION TITLE
升级专业版已付款，邮箱一直收不到登录验证码，请解决或退款

### DISCUSSION THREAD

**Discussion by WCYYYBS** [getlantern/lantern-forum-cn#560] [Q&A]
本人于2024年6月12日购买了蓝灯专业版服务，但邮箱至今未收到验证信息，无法登录，请解决或退款。邮箱为：364594461@qq.com；付款成功截图附上
![a8061aa205bdfad32d97146d74bdc02](https://github.com/getlantern/lantern-forum-cn/assets/172497914/4ac1b55a-4cce-46b4-8744-20864918d0d7)

**Comment by WCYYYBS** [comment]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@WCYYYBS 
您好. 好的, 我这里帮您要验证码. 请您稍等.

**Reply by WCYYYBS** [reply]
已经三日过去了，并未收到任何验证码的邮件，请发激活码到邮箱，请迅速解决问题，已影响正常工作

**Reply by WCYYYBS** [reply]
> @WCYYYBS 您好. 好的, 我这里帮您要验证码. 请您稍等.

10日过去了，并未收到任何验证码的邮件，请发激活码到邮箱，请迅速解决问题，已影响正常工作

**Reply by MaxMKMax** [reply]
@WCYYYBS 我刚刚又帮您问了. 等今晚客服的答复. 不好意思让您久等了.

**Comment by MaxMKMax** [comment]
@WCYYYBS 您好. 这是您专属的激活码. 谢谢您的耐心等待.
P9WP6-8QJ22-RFCMX-CJ44K-PKYTB
